### PR TITLE
completed images and input

### PR DESCRIPTION
**Pull request by GarvG**
Completed My Input and Images Please review PR

### CODE REVIEW

File: images.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    width: 70%;
+}
+.RES-IMG-ROUND
+{
+    border-radius: 50%;
+   
+    height: auto;
+    width: 20%;
+    object-fit:cover;
+}
```
Comment (sudipkundu999) [inline images.css]: responsive image means the image changes its size
when the screen size is changed maintaining its original properties.
for this we do - width: 100%; height: auto;
and apply this property to the image

Comment (GarvG) [reply images.css]: Done

**Review by sudipkundu999 — APPROVED** [review]
Try to maintain consistent spacing! (use Prettier)
Try to use better naming of classes eg. image-container.
Good to merge.

File: Images.Html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                
+                <div class="center-container">
+                    <h3 class="headers">RESPONSIVE IMAGE<h3>
+                    <div class="IMAGE">
+                        <img src="crazy.jpg" alt="" class="RES-IMG">
+                    </div>
+                
+                    <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=lucario&wt=none&l=auto&width=819.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522IMAGE%2522%253E%250A%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520class%253D%2522RES-IMG%2522%253E%250A%2520%2520%253C%252Fdiv%253E%250A"
+  style="width: 820px; height: 245px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (dev-prabhat) [inline Images.Html:45]: Try to add external css

**Review by dev-prabhat — CHANGES_REQUESTED** [review]

File: Images.Html (hunk 2 above)
Comment (GarvG) [reply Images.Html:45]: Okay I will Take care from next Time

**Pull request merged**